source/2019-10-11-the-ember-times-issue-119.md

```diff
@@ -54,13 +55,20 @@ Read the [full blog post](https://embersteps.com/promises-and-ui-states-in-ember
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [The Evolution of Ember.js at Intercom 📖](https://www.youtube.com/watch?v=NoCxHTxpmSQ)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+At [JSDayIE 2019](https://www.jsday.org/), [Gavin Joyce (@GavinJoyce)](https://github.com/GavinJoyce) presented a talk on how the adoption of Ember has helped Intercom grow over the years since 2014.
```

Comment:
```suggestion
At [JSDayIE 2019](https://www.jsday.org/), [Gavin Joyce (@GavinJoyce)](https://github.com/GavinJoyce) presented a talk on how the adoption of Ember has helped [Intercom](https://www.intercom.com/) grow over the years since 2014.
```